V3SampleApp/AppDelegate.swift

```diff
@@ -23,6 +23,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         
         LogConfig.formatters = [PrefixLogFormatter(prefixes: [.info: "𝒊", .debug: "🛠", .warning: "⚠️", .error: "🚨"])]
+        
+        LogConfig.showThreadName = false
+        LogConfig.showDate = false
+        LogConfig.showFunctionName = false
```

Comment:
But those are super helpful during debugging :(

Reply:
Really? It was too much noise for me :)